### PR TITLE
SLE15: Merge some stuff from Lenovo branch

### PR DESCRIPTION
**Pull request by Obihoernchen**
From https://github.com/jjohnson42/xcat-core/tree/lenovobuild2.14

### CODE REVIEW

File: xCAT-server/share/xcat/install/sles/compute.sle15.tmpl [hunk 1]
```diff
@@ -45,7 +45,9 @@
     </partitioning>
     <add-on>
         <add_on_products config:type="list">
-            #INSTALL_SOURCES# 
+            <listentry><media_url>http://XCATNEXTSERVERHOOK/install/#TABLE:nodetype:THISNODE:os#/x86_64/2</media_url><product>sle-module-basesystem</product><product_dir>/Module-Basesystem</product_dir></listentry>
+            <listentry><media_url>http://XCATNEXTSERVERHOOK/install/#TABLE:nodetype:THISNODE:os#/x86_64/2</media_url><product>sle-module-hpc</product><product_dir>/Module-HPC</product_dir></listentry>
+            <listentry><media_url>http://XCATNEXTSERVERHOOK/install/#TABLE:nodetype:THISNODE:os#/x86_64/2</media_url><product>sle-module-server-applications</product><product_dir>/Module-Server-Applications</product_dir></listentry>
         </add_on_products>
```
Comment (cxhong) [inline xCAT-server/share/xcat/install/sles/compute.sle15.tmpl]: is there reason why you only wants those three?

Comment (Obihoernchen) [reply xCAT-server/share/xcat/install/sles/compute.sle15.tmpl]: No not really. I just copied this from Lenovo branch. We can include all, but usually this should be sufficient for most compute nodes images I guess.

Comment (cxhong) [reply xCAT-server/share/xcat/install/sles/compute.sle15.tmpl]: I think we like to include all.  Can u put back `#INSTALL_SOURCES#` ?

Comment (Obihoernchen) [reply xCAT-server/share/xcat/install/sles/compute.sle15.tmpl]: done

Comment (cxhong) [reply xCAT-server/share/xcat/install/sles/compute.sle15.tmpl]: Thanks

**Pull request merged**